contents/docs/_snippets/how-to-capture-group-events.mdx

```diff
@@ -0,0 +1,52 @@
+
+In our [JavaScript web SDK](/docs/libraries/js), any event captured after calling `posthog.group` will be associated with that group.
+
+To capture group events using our server-side SDKs, you use the same `posthog.capture` method you would usually use, however we recommend using the group ID for the distinct ID argument.
+
+Below are code examples on how to capture group events using our various SDKs:
+
+<MultiLanguage>
+
+```js-web
+// All events for that session will be associated with company `company_id_on_your_db` (and with the user who submitted the event)
+posthog.group('company', 'company_id_in_your_db');
+
+posthog.capture('company_upgraded_plan')
+```
+
+```python
+posthog.capture('company_id_in_your_db', 
+    'company_upgraded_plan', 
+)
+```
+
+```go
+client.Enqueue(posthog.Capture{
+    DistinctId: "company_id_in_your_db",
+    Event: "company_upgraded_plan",
+})
+```
+
+```node
+posthog.capture({
+    event: 'company_upgraded_plan',
+    distinctId: 'company_id_in_your_db',
+})
+```
+
+```php
+PostHog::capture(array(
+    'distinctId' => 'company_id_in_your_db',
+    'event' => 'company_upgraded_plan'
+));
+```
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "event": "company_upgraded_plan",
+    "distinct_id": "company_id_in_your_db",
+}' <ph_instance_address>/capture/
+```
+
+</MultiLanguage>
```

Comment:
I've read the context and I think this should come with many, many warnings. 

This isn't really group analytics, but treating a user like a group. This won't work nicely with existing group analytics because it won't contain `$group` properties (which should be included in this example).

A majority of use cases will be the standard group analytics coming from a user, so we should add a lot of warnings here saying "ONLY DO THIS IF YOU DON'T WANT TO RELATE THESE EVENTS TO A USER" with examples of when you would do that (such as the repos you talk about).